### ISSUE TITLE
How to a Sponsor Banner under Table of Contents in Docusuarus?

### ISSUE THREAD

**Issue by Chetan11-dev** [facebook/docusaurus#9124]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

Sorry for asking here. But I was not able to find it's solution anywhere. 
So basically, I want to add banner similar to show in image. Could you share doc link where the how to is written?
![Frame 8](https://github.com/facebook/docusaurus/assets/53407137/d7409758-2e96-4e10-be32-2075447917e9)

### Steps to reproduce

None

### Expected behavior

None

### Actual behavior

None

### Your environment

- Public source code:
- Public site URL:
- Docusaurus version used:
- Environment name and version (e.g. Chrome 89, Node.js 16.4):
- Operating system and version (e.g. Ubuntu 20.04.2 LTS):

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by Josh-Cena** [comment]
Please, you're already aware that this is not the appropriate channel. Let's continue in https://github.com/facebook/docusaurus/discussions/9123 instead.
